backend/app/views/spree/admin/prices/_table.html.erb

```diff
@@ -0,0 +1,53 @@
+<%= paginate prices, theme: "solidus_admin" %>
+
+<table class="index prices">
+  <colgroup>
+    <col style="width: 15%" />
+    <col style="width: 35%" />
+    <col style="width: 35%" />
+    <col style="width: 15%" />
+  </colgroup>
+
+  <thead data-hook="prices_header">
+    <tr>
+      <th><%= Spree::Variant.model_name.human %> </th>
+      <th><%= Spree::Price.human_attribute_name(:is_default) %></th>
+      <th><%= Spree::Price.human_attribute_name(:amount) %></th>
+      <th><%= Spree::Price.human_attribute_name(:currency) %></th>
+      <th class="actions"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+  <% prices.each do |price| %>
+    <tr data-hook="prices_row" class="<%= "deleted" if price.deleted? %> <%= cycle('odd', 'even')%>">
+      <td class="align-center">
+        <% if price.variant.is_master? %>
+          <%= t(".master_variant") %>
+        <% else %>
+          <%= price.variant.options_text %> </td>
+        <% end %>
+      <td class="align-center">
+        <% if price.is_default? %>
+          <span class="state complete"> <%= Spree.t(:say_yes) %> </span>
+        <% else %>
+          <span class="state canceled"> <%= Spree.t(:say_no) %> </span>
+        <% end %>
+      </td>
+      <td class="align-center"><%= price.money.to_html %></td>
+      <td class="align-center"><%= price.currency %></td>
+      <td class="actions">
+        <% if can?(:update, price) %>
+          <%= link_to_edit(price, :no_text => true) unless price.deleted? %>
+        <% end %>
+        <% if can?(:destroy, price) %>
+          &nbsp;
+          <%#= link_to_delete(price, :no_text => true) unless price.deleted? %>
```

Comment:
Yes it is - there's no route for deletion yet. I should add that I guess :sheep: .